clang/test/Sema/inline-static-var.c

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -std=c11 -Wno-unused-variable -Wno-deprecated-non-prototype -Wno-inline -pedantic-errors -verify %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -std=c11 -pedantic-errors -verify %s
```
I think you can drop the other `-Wno-` lines because those aren't enabled by default anyway.